packages/lintcstubs/lintcstubs.0.4.7/opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "dune" {>= "3.0"}
   "ocaml" {>= "4.13"}
   "lintcstubs-gen" {= version}
-  "goblint" {>= "2.1.0"}
+  "goblint" {>= "2.1.0" & < "2.2.0"}
```

Comment:
@edwintorok Goblint's library interface hasn't been designed for external use and regularly breaks between releases, so you'll probably want to adapt lintcstubs. Previous Goblint releases are incompatible with OCaml 5.0, which also means lintcstubs is, which is unfortunate (OCaml 5 C stubs have to be analyzed under OCaml 4.14 instead).

Reply:
No problem, I'll add some tighter bounds in my repo, thanks for fixing it up. In general I intend to track the latest release of goblint, I think master of goblint already works with OCaml 5.

Reply:
https://github.com/edwintorok/lintcstubs/tree/goblint2.2.1-compat I've done an experiment to upgrade to latest goblint on this branch, there are some new warnings from goblint, but seems to work so far and the diff is fairly small: https://github.com/edwintorok/lintcstubs/commit/ce8150a60e7a013c33a93090b80596d4a1d0b769